### DISCUSSION TITLE
Zombies in kitodo_config.properties

### DISCUSSION THREAD

**Discussion by aetherfaerber** [kitodo/kitodo-production#6003] [Q&A] [answered]
In der [kitodo_config.properties](https://github.com/kitodo/kitodo-production/blob/master/Kitodo/src/main/resources/kitodo_config.properties#L437) können mehrere Optionen für die Anzahl vom Zombieprozessen, sowohl erfolgreich abgeschlossene als auch fehlgeschlagene Threads, konfiguriert werden.
Hierbei findet sich bereits der Kommentar „choose these values as restrictive as possible“ mit der Erklärung „zombie processes still occupy all their ressources“. Tatsächlich stellen wir einen durchaus erhöhten Ressourcenverbrauch durch diese Prozesse fest. Daher stellt sich die Frage: Wozu werden diese Prozesse überhaupt im RAM gehalten, anstatt sie sofort zu entsorgen und die Ressourcen wieder frei zu geben?

**Comment by henning-gerhardt** [comment, marked as answer]
Erfolgreich ausgeführte Verarbeitungsprozesse / Jobs im Taskmanager von Kitodo.Production sollten in der Tat schnell wieder gelöscht werden. Fehlgeschlagene sollten aber eine gewisse Zeit sichtbar sein, da dies meistens die einzige Möglichkeit überhaupt ist, diese zu erkennen, sofern der fehlgeschlagene Job im Log von Kitodo.Production selbst keine eindeutige Fehlermeldung hinterlassen hat.
